uwblueprint/childrens-aid-society
# Capitalize transport methods

## Notion ticket link

[Change transportation methods to all uppercase](https://www.notion.so/uwblueprintexecs/Change-transportation-methods-to-all-uppercase-b475eb7040d040b9ab5e55bcf667bc13)



## Implementation description
* Capitalize transportation methods
* Rename `models/transportation_methods.py` -> `models/transportation_method.py`



## Steps to test
1. Reset database: `docker exec -it cas_py_backend /bin/bash -c "flask db downgrade base && flask db upgrade"`
2. Exec into database container: `docker exec -it cas_db psql -U postgres -d cas`
3. Verify transportation methods are all capitalized: `SELECT * FROM transportation_methods;`
```
 id | transportation_method 
----+-----------------------
  1 | AGENCY DRIVER
  2 | KIN PROVIDER
  3 | FOSTER PARENT
```


## What should reviewers focus on?
* only updating the migration file directly because it shouldn't affect any future revs since it's just a change to seed data - please don't do this normally!!!


## Checklist
